src/sick_mrs1000_communication.cpp

```diff
@@ -42,25 +42,41 @@ SickMrs1000Communication::SickMrs1000Communication(const std::string &hostname,
                                                    ScanAndCloudParser* parser)
 : SickTimCommonTcp(hostname, port, timelimit, parser),
   scan_and_cloud_parser_(parser),
-  cloud_pub_(nh_.advertise<sensor_msgs::PointCloud2>("cloud", 300)),
-  diagnosed_cloud_publisher_(cloud_pub_, diagnostics_,
-    diagnostic_updater::FrequencyStatusParam(&expectedFrequency_, &expectedFrequency_, 0.1, 10),
-    diagnostic_updater::TimeStampStatusParam(-1, 1.3 * 1.0/expectedFrequency_ - config_.time_offset)
-  )
+  cloud_pub_(nh_.advertise<sensor_msgs::PointCloud2>("cloud", 300))
 {
-  expectedFrequency_ = 12.5; // by data sheet 50 Hz, 4 x 12,5 Hz
+
+  updater_mrs_1000_ = new marble::DiagnosticUpdater("/"+namespace_+"/"+"cloud", nh_);
+  marble::diagnostics::FrequencyParams warning_freq_params;
```

Comment:
Same comment for loading all the of the `FrequencyParams` fields, and them not being dynamically reconfigurable.

Reply:
Noted, just keeping expected fps and fps tolerance parameters in line with what was done in gscam, realsense and velodyne drivers

Reply:
So this is still only exposing a subset of the available parameters. Is there a reason for doing that? Why not just load the parameters for the entire `FrequencyParams` struct and pass that in?